sitemap.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset
+      xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
+            http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+<!-- created with Free Online Sitemap Generator www.xml-sitemaps.com -->
+
+
+<url>
+  <loc>https://nathandh.github.io/mywebclass-simulation/</loc>
+  <lastmod>2023-03-13T00:58:10+00:00</lastmod>
+  <priority>1.00</priority>
+</url>
+<url>
+  <loc>https://nathandh.github.io/mywebclass-simulation/content.html</loc>
+  <lastmod>2023-03-13T00:58:10+00:00</lastmod>
+  <priority>0.80</priority>
+</url>
+<url>
+  <loc>https://nathandh.github.io/mywebclass-simulation/story.html</loc>
+  <lastmod>2023-03-13T00:58:10+00:00</lastmod>
+  <priority>0.80</priority>
+</url>
+<url>
+  <loc>https://nathandh.github.io/mywebclass-simulation/privacy.html</loc>
+  <lastmod>2023-03-13T00:58:10+00:00</lastmod>
+  <priority>0.80</priority>
```

Comment:
Question: From documentation it seems the default priority applied to an entry in a SiteMap is 0.5: https://www.sitemaps.org/protocol.html

It was also noted that applying the same priority is not useful for a URL. 
In this case, should we remove the priority here, or adjust it to prioritize better (rather than the last 3 being all .80).

Reply:
I think I have made all the changes that you requested. Let me know if there is anything else. Thanks.

Reply:
Oh and before you approve, could you double-check to make sure that the files are in the right location...if not feel free to move them. Thanks!